### DISCUSSION TITLE
Flopy load recharge package

### DISCUSSION THREAD

**Discussion by mahaderony** [modflowpy/flopy#1482] [Q&A] [answered]
I am having issue while loading an existing modflow model in flopy. if I import recharge package the model does not run and  in the "list" file it says " CANNOT OPEN Carrier.ddn ON UNIT  19 BECAUSE UNIT IS ALREADY BEING USED" by the rch package

![github](https://user-images.githubusercontent.com/54357731/182943933-5a279808-2203-4b64-99cf-ab3063a80fc9.PNG)

**Comment by jlarsen-usgs** [comment, marked as answer]
You can either change the unit number you are using for your drawdown in the OC and name files or you can update the recharge unit number in flopy.

Ex 1. change unit number of drawdown
In the OC input file 
DRAWDOWN SAVE UNIT 72

In the Name file

DATA(BINARY)    72    Carrier.ddn


Ex 2, change the recharge package unit number in FloPy

```python
ml = flopy.modflow.Modflow.load("my model")
ml.rch.unit_number = [15]
```

**Reply by mahaderony** [reply]
It worked!
